src/ACF/Block_Config.php

```diff
@@ -43,6 +43,11 @@ public function init() {
 	 * @return $this
 	 */
 	public function set_block( Block $block ): Block_Config {
+		$attr = $block->get_attributes();
+		if ( empty( $attr[ 'render_callback' ] ) && empty( $attr[ 'render_template' ] ) ) {
+			$block->set( 'render_callback', [ 'Tribe\Libs\ACF\Block_Renderer', 'render_template' ] );
```

Comment:
Instead of a static method, what would you say to a closure that does a `do_action()`? The `Block_Renderer` class could then be hooked into that action. 

```
$block->set( 'render_callback', function( ... $args ) {
  do_action( 'tribe/project/block/render', ... $args );
  // or: echo apply_filters( 'tribe/project/block/render', '', ... $args )
} );
```

Reasons:
1. It avoids the static method. We can have a proper instance of the class that supports DI, overrides, etc.
2. We can hook into it for other unspecified reasons in the future

Reply:
Great idea. Thanks